lib/cc/cli/path_fingerprint.rb

```diff
@@ -0,0 +1,33 @@
+require "digest/md5"
+
+module CC
+  module CLI
+    class PathFingerprint
+      def initialize(output)
+        @output = output
+      end
+
+      def compute
+        md5 = Digest::MD5.new
+        md5 << key
+        md5.hexdigest
+      end
+
+      def key
+        "#{path}|#{check_name}"
```

Comment:
Hmm I thought it would do some work the help with collisions, but it doesn't. I think `|` is a little naive of a separator (especially since it's so common in source listings). WDYT of using `\0` instead?